src/spec-node/featuresCLI/publishCommandImpl.ts

```diff
@@ -0,0 +1,92 @@
+import { fetchRegistryAuthToken, HEADERS } from '../../spec-configuration/containerFeaturesOCI';
+import { request } from '../../spec-utils/httpRequest';
+import { Log, LogLevel } from '../../spec-utils/log';
+
+const semverCompare = require('semver-compare');
```

Comment:
This can probably be converted to `import ...` statements, like above

Reply:
Done